prompttools/testing/test_response_quality.py

```diff
@@ -0,0 +1,51 @@
+from prompttools.structures.prompt import PromptTest
+from prompttools.structures.config import get_model_params_config
+
+from langchain import HuggingFaceHub, PromptTemplate, LLMChain
+from prompttools.utils import similarity
+
+
+def test_response_hugging_face() -> None:
+    """
+    Model repos are via Hugging Face. So pytorch_cos_sim is
+    used for computing distance.
+    """
+
+    model_params = get_model_params_config()
+
+    question = model_params.question
+    template = """Question: {question}
+
+    Answer: """
+
+    prompt = PromptTemplate(template=template, input_variables=["question"])
+    expected = model_params.expected
+    hf_repo_ids = model_params.hf_repo_ids
+    temperatures = model_params.temperatures
+    max_lengths = model_params.max_lengths # 17 for deterministic response: George Washington
+
+    tests = []
+
+    for repo_id in hf_repo_ids:
```

Comment:
To create requests for every argument combination, we use experiments 

Here we gather all arguments:
https://github.com/hegelai/prompttools/blob/main/prompttools/experiment/openai_chat_experiment.py#L17-L53

And here we take their possible combos:
https://github.com/hegelai/prompttools/blob/main/prompttools/experiment/experiment.py#L126-L132

To set up this test, create an experiment with the relevant parameters instead. You would need to make a `HuggingFaceHubExperiment`, and make sure you cover all model args, not just `repo_id`,`temperature` and `max_length`.

Reply:
Thanks for the feedback @steventkrawczyk . I'm making a few commits now. Will tag for review once I think it's in a good place.